WindowsStubs/WindowsStubs/TaskUtil.cpp

```diff
@@ -0,0 +1,85 @@
+#include <windows.h>
+#include <cstdio>
+#include <tchar.h>
+#include <iostream>
+#include <string>
+#include <cctype>
+#include <algorithm>
+#include <memory>
+
+
```

Comment:
What do you think about placing all those utilities from WindowsStubs in a namespace?